### DISCUSSION TITLE
Julia for the planetary computer

### DISCUSSION THREAD

**Discussion by jdiaz97** [microsoft/PlanetaryComputer#169] [Ideas]
Julia is a [very fast](https://julialang.org/assets/images/benchmarks.svg) programming language that is used for numerical computing and data science. I think it would be a good idea to have environments that can run Julia. 

It has enough packages to work very well with the geospatial data and multithreading is extremely easy to do.

**Comment by TomAugspurger** [comment]
Thanks! I'm guessing you're specifically looking a Julia environment to be added to the [Planetary Computer Hub](https://planetarycomputer.microsoft.com/docs/overview/environment/)? You can already use Julia from your own compute deployed Azure to query our STAC API and read the data we host.

Do you know of any STAC clients written in Julia? I don't see anything on https://www.stacindex.org/ecosystem, but a quick search turned up https://github.com/JuliaClimate/STAC.jl.

As for getting a Julia environment added to our Hub, it might take a while. Our environment management is all conda-based and builds on https://github.com/pangeo-data/pangeo-docker-images. I'm not familiar with packaging in Julia or what the state of it is on conda-forge.
